.wordlist.txt

```diff
@@ -209,3 +209,11 @@ Diátaxis
 reStructuredText
 localhost
 HTML
+Nvidia
+nvidia
```

Comment:
Remove these two entries. We should be using the same brand name everywhere, so let's use 'NVIDIA' (as it is used on their websites) everywhere.

Reply:
lowercase nvidia is how the console ouput shows it. If I remove it from the list, then the spellchecking doesn't pass. Any advice?